Knapsack-based-Cryptography.py

```diff
@@ -0,0 +1,107 @@
+# Encrypt your first name into a binary string, each letter is encoded by 5 bits. 
+# A is encoded to 00000
+# B is encoded to 00001 
+# ...
+
+# Bui Van Thanh - 20200585
+from math import gcd
+import random
+
+ID = 85 # two last digit in my student ID
+
+def encode_string(string):
```

Comment:
Cái hàm encode_string, thấy Hoàng Huy có hàm tương tự nhưng viết ngắn gọn hơn nè:
def encode_to_bits(name):
    bit_string = ''
    for i in name:
        bit_string += '{0:05b}'.format(ord(i.capitalize()) - ord('A'))
    return bit_string

Reply:
Ok, I'll fix it